services/frontend-service/src/ui/utils/store.tsx

```diff
@@ -363,10 +364,10 @@ export const useApplicationLock = (lockId: string) =>
         )?.name,
     } as DisplayLock);
 
-export const useLock = (lockId: string) =>
+export const useLock = (lockId: string): DisplayLock | undefined =>
```

Comment:
Also the same comment as before? 

Additionally, to me, this function has no usages anywhere. do we need to keep it?

Reply:
done, removed it.